requestor/service_manager/yapapi_connector.py

```diff
@@ -7,8 +7,9 @@
 
 
 class YapapiConnector():
-    def __init__(self, executor_cfg: dict):
+    def __init__(self, executor_cfg: dict, exception_handler):
```

Comment:
```suggestion
    def __init__(self, executor_cfg: dict, exception_handler: 'Callable[[Exception], Any]'):
```

Reply:
I fixed this & few more things in 041fc04